### DISCUSSION TITLE
Error after initialization 初始化后提示错误

### DISCUSSION THREAD

**Discussion by susanforme** [alex8088/electron-vite#113] [General]
![image](https://user-images.githubusercontent.com/51266600/220515439-bdff8577-7440-4647-9096-0a6e7a00ac75.png)
The type definition file for "debug" could not be found.

The program includes this file because:

Entry point to the implicit type library "debug"

**Comment by alex8088** [comment]
模块是否已安装？

**Reply by susanforme** [reply]
安装了,跑buidling 一样有这个错,看样子是没导入node_modules中@types包,有什么解决办法吗?
![image](https://user-images.githubusercontent.com/51266600/220533565-85c09767-a041-4df9-bd53-28c6a1571670.png)
